packages/fleather/lib/src/widgets/controller.dart

```diff
@@ -26,6 +33,14 @@ class FleatherController extends ChangeNotifier {
   TextSelection get selection => _selection;
   TextSelection _selection;
 
+  List<RemoteSelection> get remoteSelections => _remoteSelections;
```

Comment:
Isn't it better to pass `remoteSelections` directly to widget (Editor or Field) instead of controller?

Reply:
I think it's more natural to provide selections in `build` like other widgets. But if you're sure that we're gonna need some optimizations in future I'm ok with the way it's now. @cgestes 
Also what do you think about this? @amantoux

Reply:
? I'am not sure I am following you.
What changes often seems to often be provided by a controller. I dont know what is better, the whole infrastructure was there to support that usecase easily in fleather. (it's already done for the local selection)